frontend/src/metabase/core/components/AutocompleteInput/AutocompleteInput.unit.spec.tsx

```diff
@@ -5,9 +5,27 @@ import AutocompleteInput from "./AutocompleteInput";
 
 const OPTIONS = ["Banana", "Orange", "Mango"];
 
-const setup = ({ value = "", onChange = jest.fn() } = {}) => {
+interface setupProps {
```

Comment:
PascalCase for types, e.g. `SetupProps`. But in this case, would it make sense to use the props type, e.g. `AutocompleteInputProps` or `Partial<AutocompleteInputProps`? You could then just spread it to the component